src/pt/randomscan/src/eu/kanade/tachiyomi/extension/pt/randomscan/LuraToon.kt

```diff
@@ -113,60 +155,50 @@ class LuraToon : HttpSource(), ConfigurableSource {
             }
     }
 
+    override fun chapterListParse(response: Response) = throw UnsupportedOperationException()
+
     fun chapterListParse(manga: SManga, response: Response): List<SChapter> {
         if (response.code == 404) {
             throw Exception("Capitulos não encontrados, tente migrar o manga, alguns nomes da LuraToon mudaram")
         }
 
-        val comics = response.parseAs<Manga>()
+        val comics = response.parseAs<MangaDTO>()
 
         return comics.caps.sortedByDescending {
             it.num
         }.map { chapterFromElement(manga, it) }
     }
 
-    private fun chapterFromElement(manga: SManga, capitulo: Capitulo) = SChapter.create().apply {
+    private fun chapterFromElement(manga: SManga, capitulo: CapituloDTO) = SChapter.create().apply {
         val capSlug = capitulo.slug.trimStart('/')
         val mangaUrl = manga.url.trimEnd('/').trimStart('/')
-        setUrlWithoutDomain("/api/obra/$mangaUrl/$capSlug")
+        url = "$mangaUrl/$capSlug"
```

Comment:
This shouldn't work unless you create the API call in pageListRequest. Has this been properly tested?

Reply:
Yes, all page requests will be made through pageListRequest in the future. They changed the code, and everything is obfuscated, which is making it difficult to adjust. This PR was made to fix the webview of the work so that readers at least have something to read. I created a debug APK, and it’s working fine, except for the pages that still aren’t functioning.